modules/nextflow/src/test/groovy/nextflow/scm/AssetManagerTest.groovy

```diff
@@ -72,6 +72,18 @@ class AssetManagerTest extends Specification {
         AssetManager.root = tempDir.root.toFile()
     }
 
+    // Helper method to grab the default brasnch if set in ~/.gitconfig
+    String getUserDefaultBranch() {
```

Comment:
```suggestion
    String getDefaultBranch() {
```

Reply:
The main info is that it's the default branch

Reply:
Done.
I used the `getUserDefaultBranch` name to avoid confusion with the existing `getDefaultBranch` on the manifest object:
```groovy
holder.manifest.getDefaultBranch()
```
... but the risk of confusion is very minimal.

Reply:
oh, entering paranoid mode and using `getLocalDefaultBranch`? 😆

Reply:
great!